client/asset/dcr/dcr.go

```diff
@@ -392,7 +392,7 @@ func (dcr *ExchangeWallet) Balance() (*asset.Balance, error) {
 		ab := &balances.Balances[i]
 		if ab.AccountName == dcr.acct {
```

Comment:
Just seeing this, do we also limit funding orders to unspent outputs in `dcr.acct`? If not, we might be understating the available balance as other accounts utxos may be used when funding orders.

Reply:
That's true. Do we allow spending from all accounts, or do we limit all activity to a single account?

If we limit activity, the user would have to send themselves the initial funds (to an DEX account address) in order to trade.

Reply:
> the user would have to send themselves the initial funds (to an DEX account address) in order to trade

That would be a good feature IMO.